debian/control

```diff
@@ -0,0 +1,193 @@
+Architecture: any
+Depends: ${shlibs:Depends}, 
+	 ${misc:Depends},
+	 python-gobject-2,
+	 libx11-6:amd64,
+	 libgl1-mesa-glx:i386,
+	 python-repoze.lru,
+	 libpython2.7-minimal:amd64,
+	 libxau6:amd64,
+	 perl,
+	 libnuma1:amd64,
+	 cpio,
+	 libxau6:i386,
+	 libc6-i386,
+	 python-ubuntu-sso-client,
+	 libgl1-mesa-glx:amd64,
+	 mawk,
+	 mime-support,
+	 libpython2.7-stdlib:amd64,
+	 tzdata,
+	 libxdmcp6:i386,
+	 libmagic1:amd64,
+	 libxcb1:i386,
+	 libice6:amd64,
+	 python2.7-minimal,
+	 python-zope.interface,
+	 file,
+	 libxcb1:amd64,
+	 libc6:i386,
+	 libfakeroot:amd64,
+	 liblocale-gettext-perl,
+	 python-imaging,
+	 libxdmcp6:amd64,
+	 libegl1-mesa:amd64,
+	 postgresql,
+	 libpq-dev,
+	 libical1a,
+	 libical-dev,
+	 swig,
+	 expat,
+	 tcl,
+	 tk
+Description: 
+ PBS Professional® is a fast, powerful workload manager and
+ job scheduler designed to improve productivity, optimize
+ utilization & efficiency, and simplify administration for
+ HPC clusters, clouds and supercomputers.
+ .
+ This package is intended for a server host. It includes all
+ PBS Professional components.
+
+Package: pbspro-execution
+Architecture: any
+Depends: ${shlibs:Depends}, 
+	 ${misc:Depends},
+	 python-gobject-2,
+	 libx11-6:amd64,
+	 libgl1-mesa-glx:i386,
+	 python-repoze.lru,
+	 libpython2.7-minimal:amd64,
+	 libxau6:amd64,
+	 perl,
+	 libnuma1:amd64,
+	 cpio,
+	 libxau6:i386,
+	 libc6-i386,
+	 python-ubuntu-sso-client,
+	 libgl1-mesa-glx:amd64,
+	 mawk,
+	 mime-support,
+	 libpython2.7-stdlib:amd64,
+	 tzdata,
+	 libxdmcp6:i386,
+	 libmagic1:amd64,
+	 libxcb1:i386,
+	 libice6:amd64,
+	 python2.7-minimal,
+	 python-zope.interface,
+	 file,
+	 libxcb1:amd64,
+	 libc6:i386,
+	 libfakeroot:amd64,
+	 liblocale-gettext-perl,
+	 python-imaging,
+	 libxdmcp6:amd64,
+	 libegl1-mesa:amd64,
+	 libical1a,
+	 swig,
+	 expat,
+	 tcl,
+	 tk
+Description: 
+ PBS Professional® is a fast, powerful workload manager and
+ job scheduler designed to improve productivity, optimize
+ utilization & efficiency, and simplify administration for
+ HPC clusters, clouds and supercomputers.
+ .
+ This package is intended for an execution host. It does not
+ include the scheduler, server, or communication agent. It
+ does include the PBS Professional user commands.
+
+Package: pbspro-client
+Architecture: any
+Depends: ${shlibs:Depends}, 
+	 ${misc:Depends},
+	 python-gobject-2,
+	 libx11-6:amd64,
+	 libgl1-mesa-glx:i386,
+	 python-repoze.lru,
+	 libpython2.7-minimal:amd64,
+	 libxau6:amd64,
+	 perl,
+	 libnuma1:amd64,
+	 cpio,
+	 libxau6:i386,
+	 libc6-i386,
+	 python-ubuntu-sso-client,
+	 libgl1-mesa-glx:amd64,
+	 mawk,
+	 mime-support,
+	 libpython2.7-stdlib:amd64,
+	 tzdata,
+	 libxdmcp6:i386,
+	 libmagic1:amd64,
+	 libxcb1:i386,
+	 libice6:amd64,
+	 python2.7-minimal,
+	 python-zope.interface,
+	 file,
+	 libxcb1:amd64,
+	 libc6:i386,
+	 libfakeroot:amd64,
+	 liblocale-gettext-perl,
+	 python-imaging,
+	 libxdmcp6:amd64,
+	 libegl1-mesa:amd64,
+	 libical1a,
+	 swig,
+	 expat,
+	 tcl,
```

Comment:
Do you want tk lister here as well? If not, you should probably remove the comma.